core/src/main/java/io/grpc/internal/ClientStream.java

```diff
@@ -46,7 +46,7 @@
    * period until {@link ClientStreamListener#closed} is called. This method is safe to be called
    * at any time and multiple times.
    *
-   * @param reason must be one of Status.CANCELLED or Status.DEADLINE_EXCEEDED
+   * @param reason must have Code CANCELLED, DEADLINE_EXCEEDED, INTERNAL, or UNKNOWN
```

Comment:
I have made the update, but it looks pretty bad.  If you like it better this way, I can keep it.